Completions are working again - ready for review (#30)
sourcegraph/emacs-cody: CONTRIBUTING.org

* Contributing to the Sourcegraph Cody Emacs Plugin

Thank you for your interest in contributing to Sourcegraph! The goal of this
document is to provide a high-level overview of how you can contribute to the
Cody Emacs Plugin.

** Issues / Bugs 

New issues and feature requests can be filed through
our [issue tracker](https://github.com/sourcegraph/emacs-cody/issues/new/choose).

** Development

*** Prerequisites

- Ensure that you have node.js installed.
  - Current min version is 20.4.0.
- Set up the Cody agent local build environment.
  - Clone `https://github.com/sourcegraph/cody` in a sibling directory.
    - Install its dependencies. The easiest way
      is [with `asdf`](https://github.com/sourcegraph/cody/blob/main/doc/dev/index.md).
      If not using `asdf`, you just need to install the dependency versions listed
      in the `.tool-versions` file in that repository.
    - From the root directory of the repository, `cd ./agent && pnpm install && pnpm build`
- Run `node ./scripts/build.js` to rebuild the agent and copy the distribution.

*** Dev loop

Once you have it working, working, then any time you make changes to (or

> update) the agent in the sibling `cody` directory, you can re-run `node ./scripts/build.js`

Author: Clarified, and also took a closer look at the whole file.

to rebuild and re-copy the distribution files into `./dist/`

*** Tree-sitter grammars

> Cody uses (tree-sitter)[https://tree-sitter.github.io/tree-sitter/] for improving

Reviewer: org mode link syntax is `[[link][description]]`
```suggestion
Cody uses [[https://tree-sitter.github.io/tree-sitter/][tree-sitter]] for improving
```

> Cody uses (tree-sitter)[https://tree-sitter.github.io/tree-sitter/] for improving

Author: Fixed all the links.

> its responses in various ways. You should consider the WASM files for at least the
> languages you use regularly with Cody to be essential for getting the best results.

Reviewer 2: should consider ... at least ... to be ... for

You have blown my stack for dependent phrases. Be direct:

```suggestion
its responses in various ways. To get the best results, the WASM files for the
languages you use regularly are essential.
```

Author: fixed
